contrib/kube-prometheus/jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet

```diff
@@ -47,12 +47,12 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
     secret:
       local secret = k.core.v1.secret;
 
-      if std.type($._config.alertmanager.config) == "object" then
-        secret.new('alertmanager-' + $._config.alertmanager.name, { 'alertmanager.yaml': std.base64(std.manifestYamlDoc($._config.alertmanager.config)) })
+      if std.type($._config.alertmanager.config) == 'object' then
+        secret.new('alertmanager-' + $._config.alertmanager.name, { 'alertmanager.yaml': std.base64(std.manifestYamlDoc($._config.alertmanager.config)) }) +
+        secret.mixin.metadata.withNamespace($._config.namespace)
```

Comment:
Do you not want the trailing `,` here?

Reply:
Actually jsonnet does not allow it here, it's not actually part of the if/else at the end, but rather ending the whole if/else.

Reply:
Sorry, I saw it on the similar line in the else block ...